apps/dashboard/src/lib/components/Org/Settings/OrgSettings.svelte

```diff
@@ -54,6 +61,26 @@
     };
   }
 
+  async function handleCustomTheme() {
+    if (!hex) return;
+
+    // Create a promise that resolves after the delay
+    await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
We should probably remove this, right?

Reply:
honestly, no, whenever that picker gets dragged we update our db so if we remove this and a user drags from point a to b in 10s, we'll be sending 10 requests to supabase so i added this one to make that request on finalize